Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs

```diff
@@ -145,7 +149,14 @@ public void TryOpen(IEntity user)
                 Deny();
                 return;
             }
+
             Open();
+
+            if (user.TryGetComponent(out HandsComponent hands) && hands.Count == 2)
```

Comment:
> `hands.Count == 2`
> no hands

:thinking:

Reply:
Gazoinks, you have caught my trick to record the video.